### PR TITLE
New Additions to This List

### PR DESCRIPTION
**Pull request by letscode-17**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -65,6 +65,7 @@
 * [JDoodle - Online C Compiler IDE](https://www.jdoodle.com/c-online-compiler/)
 * [Programiz - Online C Compiler](https://www.programiz.com/c-programming/online-compiler/)
 * [SoloLearn](https://code.sololearn.com/c)
+* [InterviewBit - Online C Compiler](https://www.interviewbit.com/online-c-compiler/)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:68]:
```suggestion
* [InterviewBit - Online C Compiler](https://www.interviewbit.com/online-c-compiler/)
* [JDoodle - Online C Compiler IDE](https://www.jdoodle.com/c-online-compiler/)
* [Programiz - Online C Compiler](https://www.programiz.com/c-programming/online-compiler/)
* [SoloLearn](https://code.sololearn.com/c)
```

File: more/free-programming-playgrounds.md [hunk 2]
```diff
@@ -79,6 +80,7 @@
 * [LabStack](https://code.labstack.com/cpp)
 * [OnlineGDB](https://www.onlinegdb.com/online_c++_compiler)
 * [SoloLearn](https://code.sololearn.com/cpp)
+* [InterviewBit - Online C++ Compiler](https://www.interviewbit.com/online-cpp-compiler/)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:83]:
```suggestion
* [InterviewBit - Online C++ Compiler](https://www.interviewbit.com/online-cpp-compiler/)
* [LabStack](https://code.labstack.com/cpp)
* [OnlineGDB](https://www.onlinegdb.com/online_c++_compiler)
* [SoloLearn](https://code.sololearn.com/cpp)
```

File: more/free-programming-playgrounds.md [hunk 3]
```diff
@@ -157,6 +159,7 @@
 * [Programiz - Online Java Compiler](https://www.programiz.com/java-programming/online-compiler/)
 * [repl.it](https://repl.it) (_including a separate Java/Swing_)
 * [SoloLearn](https://code.sololearn.com/java)
+* [InterviewBit - Online Java Compiler](https://www.interviewbit.com/online-java-compiler/)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:162]:
```suggestion
* [InterviewBit - Online Java Compiler](https://www.interviewbit.com/online-java-compiler/)
* [Programiz - Online Java Compiler](https://www.programiz.com/java-programming/online-compiler/)
* [repl.it](https://repl.it) (_including a separate Java/Swing_)
* [SoloLearn](https://code.sololearn.com/java)
```

File: more/free-programming-playgrounds.md [hunk 4]
```diff
@@ -170,6 +173,7 @@
 * [JSFiddle](https://jsfiddle.net)
 * [Plunker](https://plnkr.co)
 * [SoloLearn](https://code.sololearn.com/web#javascript)
+* [InterviewBit - Online JavaScript Compiler]([InterviewBit - Online Java Compiler](https://www.interviewbit.com/online-java-compiler/))
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:176]: Fix Markdown syntax and alphabetize list a bit transfering up

```suggestion
* [InterviewBit - Online JavaScript Compiler](https://www.interviewbit.com/online-java-compiler/)
* [JSFiddle](https://jsfiddle.net)
* [Plunker](https://plnkr.co)
* [SoloLearn](https://code.sololearn.com/web#javascript)
```

File: more/free-programming-playgrounds.md [hunk 5]
```diff
@@ -237,6 +241,7 @@
 * [Repl.it - NiceDualPoint](https://repl.it/repls/NiceDualPoint#main.py)
 * [SoloLearn](https://code.sololearn.com/python)
 * [Try Python](https://trypython.jcubic.pl) - Jakub T. Jankiewicz
+* [InterviewBit - Online Python Compiler](https://www.interviewbit.com/online-python-compiler/)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:244]:
```suggestion
* [InterviewBit - Online Python Compiler](https://www.interviewbit.com/online-python-compiler/)
* [Repl.it - NiceDualPoint](https://repl.it/repls/NiceDualPoint#main.py)
* [SoloLearn](https://code.sololearn.com/python)
* [Try Python](https://trypython.jcubic.pl) - Jakub T. Jankiewicz
```

**Review by davorpa — CHANGES_REQUESTED** [review]
As Eric says... needs list alphabetization. Here some of suggestions to be applied

**Comment by letscode-17** [comment]
Hey, Thanks for response. Have created another pull-request basis on the changes suggested.

**Comment by davorpa** [comment]
Work has been continued at #6817

**Pull request closed without merging**